javascript/ql/src/semmle/javascript/security/dataflow/TaintedPathCustomizations.qll

```diff
@@ -355,6 +351,47 @@ module TaintedPath {
     }
   }
 
+  /**
+   * A sanitizer that recognizes the following pattern:
+   *   var relative = path.relative(webroot, pathname);
+   *   if(relative.startsWith(".." + path.sep) || relative == "..") {
+   *     // pathname is unsafe
+   *   } else {
+   *     // pathname is safe
+   *   }
+   */
+  class RelativePathContainsDotDotGuard extends DataFlow::BarrierGuardNode {
```

Comment:
`RelativePathContainsDotDotGuard ` -> `RelativePathStartsWithDotDotSanitizer`

The other guards are called `Sanitizer` so we might as well be consistent, and this handles `StartsWith` checks, not inclusion checks.